pkg/git/ssh_config.go

```diff
@@ -33,7 +33,10 @@ Host {{ .Host }}
     StrictHostKeyChecking no
 `
 
-var sshConfigTmpl = template.Must(template.New("ssh-config").Parse(sshConfigTemplate))
+var (
+	sshConfigTmpl            = template.Must(template.New("ssh-config").Parse(sshConfigTemplate))
+	defaultSSHConfigFilePath = "/home/pipecd/.ssh/config"
```

Comment:
Alternatively, `$HOME/.ssh/config`?

Reply:
The current one is fine to be unified with other places.

Reply:
👍